src/test-routes/media-types.ts

```diff
@@ -18,4 +18,22 @@ app.category("vanilla", () => {
       };
     }
   });
+  app.post("/mediatypes/binaryBodyTwoContentTypes", "MediaTypesTwoContentTypes", (req) => {
+    const contentType = req.headers["content-type"];
+    if (contentType === "application/json") {
+      req.expect.bodyNotEmpty();
```

Comment:
Can we check it looks like a JSON at least? To be sure you didn't send bytes by mistake?